includes/class-wc-dintero-hp-ajax.php

```diff
@@ -575,7 +575,16 @@ public function dhp_create_order() {
 					self::failed_order( $order, $transaction_id, $fail_reason );
 				} else {
 					$fail_reason = __( 'Failed updating transaction with order_id ' ) . '.';
-					self::failed_order( $order, $transaction_id, $fail_reason );
+					$order->add_order_note( $fail_reason );
+
+					$update_response_retry = self::_adapter()->update_transaction(
+						$transaction_id, $update_payload
+					);
+
+					if (is_wp_error($update_response_retry)) {
+						$fail_reason = __( 'Failed updating transaction with order_id. Will stop trying ' ) . '.';
```

Comment:
Do we need to add another note if success?

Reply:
Added